doc/whats_new.rst

```diff
@@ -127,15 +127,22 @@ Enhancements
 
    - Allow :func:`datasets.make_multilabel_classification` to output
      a sparse ``y``. By Kashif Rasul.
-     
+
    - :class:`cluster.DBSCAN` now accepts a sparse matrix of precomputed
      distances, allowing memory-efficient distance precomputation. By
      `Joel Nothman`_.
-     
+
    - :class:`tree.DecisionTreeClassifier` now exposes an ``apply`` method
      for retrieving the leaf indices samples are predicted as. By
      `Daniel Galvez`_ and `Gilles Louppe`_.
 
+   - Speed up decision tree regressor, random forest regressor, extra trees
+     regressor and gradient regression tree boosting by computing a proxy
```

Comment:
It should speed up both Gradient Boosting Regression and Gradient Boosting Classification, since it turns a classification problem into a regression problem. Maybe better to say "extra trees regressor and gradient boosting"